lib/B/Deparse.pm

```diff
@@ -3954,7 +3954,18 @@ sub loop_common {
 	} else {
 	    $ary = $self->deparse($ary, 1);
 	}
-	if (null $var) {
+        my $iter_targ = $kid->first->first->targ;
```

Comment:
I'm not sure how pedantic we want to be here, but the indentation looks off in Github because Deparse.pm is indented with tabs while the code added here is indented with spaces.

Reply:
I'm not sure either.

Usually I try to match the indenting (etc) style of the surrounding code, and I missed that this file had tabs.

Well, ...:

```
$ git show 8b7aa8e4305788f70f89b55541ae664e8ecb17e0 lib/B/Deparse.pm | perl -pe 's/\t/......../g'
commit 8b7aa8e4305788f70f89b55541ae664e8ecb17e0
Author: Nicholas Clark <nick@ccl4.org>
Date:   Mon Apr 26 11:23:33 2021 +0000

    B::Deparse now handles n-at-a-time for.

diff --git a/lib/B/Deparse.pm b/lib/B/Deparse.pm
index 2d33039a80..12f6a63670 100644
--- a/lib/B/Deparse.pm
+++ b/lib/B/Deparse.pm
@@ -52,7 +52,7 @@ use B qw(class main_root main_start main_cv svref_2object opnumber perlstring
         MDEREF_SHIFT
     );

-$VERSION = '1.58';
+$VERSION = '1.59';
 use strict;
 our $AUTOLOAD;
 use warnings ();
@@ -3954,7 +3954,18 @@ sub loop_common {
 ........} else {
 ........    $ary = $self->deparse($ary, 1);
 ........}
-........if (null $var) {
+        my $iter_targ = $kid->first->first->targ;
+        if ($iter_targ) {
+            # for my ($foo, $bar) () stores the count (less 1) in the targ of
+            # the ITER op.
+            my @vars;
+            my $targ = $enter->targ;
+            while ($iter_targ-- >= 0) {
+                push @vars, $self->padname_sv($targ)->PVX;
+                ++$targ;
+            }
+            $var = 'my (' . join(', ', @vars) . ')';
+        } elsif (null $var) {
             $var = $self->pp_padsv($enter, 1, 1);
 ........} elsif ($var->name eq "rv2gv") {
 ........    $var = $self->pp_rv2sv($var, 1);
```

it does and it doesn't. The line above my new block had a tab, but the first line after it did not.

The whole file is a inconsistent. There are parts of it with tabs, and other parts (I *guess* newer, but I didn't see a good way to check this) that have spaces.

So having discovered that, there doesn't seem to be any "right" answer, so I'm tempted to say that this part can be spaces.

Reply:
I thought we converted lines we touched to spaces, from perlhack:
```
The codebase is a mixture of tabs and spaces for indentation, and we
are moving to spaces only.  Converting lines you're patching from 8-wide
tabs to spaces will help this migration.
```